processor/transformprocessor/testdata/invalid_config_unknown_function_metric.yaml

```diff
@@ -0,0 +1,19 @@
+processors:
+  transform:
+    metrics:
+      queries:
+        - set(body, "bear") where attributes["http.path"] == "/animal"
```

Comment:
This seems like it will fail on `body` instead of the unknown function IIUC

Reply:
Ya I missed this in the copy and paste. Will fix.

Reply:
Fixed.  Now properly fails with "processor "transform" has invalid configuration: undefined function not_a_function"